src/pages/BlockDetail/BlockComp.tsx

```diff
@@ -90,188 +90,191 @@ const BlockMinerReward = ({
+        },
+      },
+      {
+        title: t('block.epoch'),
+        tooltip: t('glossary.epoch'),
+        content: localeNumberString(block.epoch),
+      },
+      {
+        title: t('block.miner_reward'),
+        tooltip: t('glossary.miner_reward'),
+        content: (
+          <BlockMinerReward
+            value={block.rewardStatus === 'pending' ? t('block.pending') : minerReward}
+            tooltip={block.rewardStatus === 'pending' ? t('block.pending_tip') : t('block.reward_sent_tip')}
+            sentBlockNumber={block.rewardStatus === 'pending' ? undefined : sentBlockNumber}
+          />
+        ),
+      },
+      {
+        title: t('block.epoch_start_number'),
+        tooltip: t('glossary.epoch_start_number'),
+        content: (
+          <BlockLinkPanel>
+            <Link to={`/block/${block.startNumber}`}>{localeNumberString(block.startNumber)}</Link>
+          </BlockLinkPanel>
+        ),
+      },
+      {
+        title: t('block.difficulty'),
+        tooltip: t('glossary.difficulty'),
+        content: handleDifficulty(block.difficulty),
+      },
+      {
+        title: t('block.block_index'),
+        tooltip: t('glossary.block_index'),
+        content: `${Number(block.blockIndexInEpoch) + 1}/${block.length}`,
+      },
+      {
+        title: t('block.nonce'),
+        tooltip: t('glossary.nonce'),
+        content: <>{`0x${new BigNumber(block.nonce).toString(16)}`}</>,
+      },
+      {
+        title: t('block.timestamp'),
+        tooltip: t('glossary.timestamp'),
+        content: `${parseSimpleDate(block.timestamp)}`,
+      },
+      {
+        title: t('block.uncle_count'),
+        tooltip: (
+          <Trans
+            i18nKey="glossary.uncle_count"
+            components={{
+              // eslint-disable-next-line jsx-a11y/control-has-associated-label, jsx-a11y/anchor-has-content
+              link1: <a href="https://docs.nervos.org/docs/basics/glossary/#uncle" target="_blank" rel="noreferrer" />,
+            }}
+          />
+        ),
+        content: `${block.unclesCount}`,
+      },
+    ]
 
-  if (isMobile) {
-    overviewItems.push(rootInfoItem)
-  }
+    if (isMobile) {
+      overviewItems.push(rootInfoItem)
+    }
 
-  return (
-    <Card>
-      <HashCardHeader title={t('block.block')} hash={blockHeightOrHash} />
-      <CardCellsLayout type="left-right" cells={overviewItems} borderTop />
-      {!isMobile && <CardCell {...rootInfoItem} className={styles.cellTransactionsRoot} />}
-    </Card>
-  )
-}
+    return (
+      <Card>
+        <HashCardHeader title={t('block.block')} hash={blockHeightOrHash} />
+        <CardCellsLayout type="left-right" cells={overviewItems} borderTop />
+        {!isMobile && <CardCell {...rootInfoItem} className={styles.cellTransactionsRoot} />}
+      </Card>
+    )
+  },
+  (pre, next) => pre.block.number === next.block.number || (pre.block.number > 0 && next.block.number === 0),
```

Comment:
Using this approach to fulfill the current requirement is not recommended because it makes the code's intention very obscure. If the intention is misunderstood by future developers, it may lead to unintended modifications and make bug troubleshooting difficult.

Furthermore, this approach cannot guarantee the fulfillment of the current requirement, as per the official documentation of React, which states:

> Wrap a component in memo to get a memoized version of that component. This memoized version of your component will usually not be re-rendered when its parent component is re-rendered as long as its props have not changed. **But React may still re-render it: memoization is a performance optimization, not a guarantee.**

Excerpt from https://react.dev/reference/react/memo#reference